README.md

```diff
@@ -10,7 +10,7 @@ The documentation is built using Markdown pages, which can be converted into a n
 
 ## Installing Hugo
 
-The documentation site uses [Hugo](https://gohugo.io/), and specifically uses features found in the extended version of Hugo, v0.53 and later.
+The documentation site uses [Hugo](https://gohugo.io/), and specifically uses features found in the extended version of Hugo, v0.71.1.
```

Comment:
Should this be in a separate PR? Also, what brought on this change? Are versions after 71.1 suitable too?

Reply:
We build the app on github actions using 71.1 but we state .53 as what we support. I believe this should match so that someone developing doesn't get confused when a feature doesn't work in older or newer version.  There's also a bug or improvement in versions earlier than 71.1 that causes the error below. 

```
Building sites … ERROR 2020/10/01 10:52:02 [en] REF_NOT_FOUND: Ref "apps/features/admin/": "~/dev/cht-docs/content/en/apps/guides/integrations/rapidpro.md:82:222": page not found
ERROR 2020/10/01 10:52:02 [en] REF_NOT_FOUND: Ref "apps/reference/app-settings/": "~/dev/cht-docs/content/en/apps/reference/api.md:701:44": page not found
Built in 808 ms
Error: Error building site: logged 2 error(s)
```

Reply:
It could be a separate PR if need be.

Reply:
Noting which version we use for CI is nice, and would be good to be excplicit about that so that people could match up their local environment accordingly. I think that is separate from the minimum version needed to build locally though. Would it work to recommended 0.71.1 since that is what we use for CI, but the minimum for local install is 0.53?

As for those errors, I think if you drop the trailing slash those references should work.

Reply:
removed from this pr